.changeset/wicked-beds-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": minor
+---
+
+We attempt simple CST error recovery for common "terminated-by" and "delimited-by" groups instead of bailing out on incomplete/invalid input
```

Comment:
nit: "terminated-by" and "delimited-by" are internal grammar concepts. I suggest using user-facing terms. Maybe something like "terminators (semicolon)" and "delimiters (brackets)" instead?